staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -221,9 +221,10 @@ enum MountPropagation {
 message Mount {
     // Path of the mount within the container.
     string container_path = 1;
-    // Path of the mount on the host. If the hostPath doesn't exist, then runtimes
-    // should report error. If the hostpath is a symbolic link, runtimes should
-    // follow the symlink and mount the real destination to container.
+    // Path of the mount on the host. Can be empty if the image field below is
+    // provided. If the host path doesn't exist and image is nil, then runtimes
+    // should report an error. If the hostpath is a symbolic link, runtimes
+    // should follow the symlink and mount the real destination to container.
     string host_path = 2;
```

Comment:
If they can't be used together, it's better to mention that host_path and the following image are mutually exclusive.

Reply:
IMO, they represent different volume sources.

Reply:
Updated the doc comment as suggested.